AC_Raíz_Certicámara_S.A.:2.15.7.126.82.147.123.224.21.227.87.240.105.140.203.236.12.crt

```diff
@@ -0,0 +1,41 @@
+# alias="AC Ra\xC3\xADz Certic\xC3\xA1mara S.A."
+# trust=CKA_TRUST_CODE_SIGNING CKA_TRUST_EMAIL_PROTECTION
+# distrust=
+# openssl-trust=codeSigning emailProtection
```

Comment:
@shrug so like, this should be deleted too, yeah? since it doesn't have `serverAuth` in the `openssl-trust` field?